docs/en/sql-reference/statements/alter/column.md

```diff
@@ -20,7 +20,6 @@ The following actions are supported:
 
 -   [ADD COLUMN](#alter_add-column) — Adds a new column to the table.
 -   [DROP COLUMN](#alter_drop-column) — Deletes the column.
--   [RENAME COLUMN](#alter_rename-column) — Renames the column.
```

Comment:
The file contains two paragraphs with RENAME COLUMN, so I suppose that the second one (at the end) is not needed. You can just simply delete it (lines 196-211) and update this list of actions (move the function title on the third position like it was before).
Same with the Russian version.

Reply:
fixed at https://github.com/ClickHouse/ClickHouse/pull/26878/commits/f9c6df4d645cf9af108b1af1ed37f0a72a3db490